daml/splice-amulet/daml/Splice/ValidatorLicense.daml

```diff
@@ -220,6 +224,17 @@ template ValidatorLivenessActivityRecord with
          void $ fetchReferenceData (ForRound with dso; round) closedRoundCid
          return ValidatorLivenessActivityRecord_DsoExpireResult
```

Comment:
if we go for introduce a new choice, then make the replaced choice "abort" if possible. Thereby reducing our testing surface.